examples/terraform-submodules/gke-local/module.tf

```diff
@@ -41,11 +37,30 @@ variable "machine_type" {
   default = "n1-standard-4"
 }
 
+variable "name" {
+  default = "agones-terraform-example"
+}
+
+variable "values_file" {
+  default = "../../../install/helm/agones/values.yaml"
```

Comment:
I'm still trying to understand why we are doing this?

Helm already defaults to it's default values, so why reset it all over again?

Or is this meant to be an example?

Reply:
This gke-local is intended to be used by developers, who have the full repo installed.
Imagine that developer wants to add some new one, or change its default, for example starting line:
```
agones:
  featureGates: "Example"
```
That way by changing local `values.yaml` he is able to use terraform with local file.
By the way it seems that I forgot about the option to `--set ` this parameters in Terraform for instance `--set gameservers.minPort=1000`

Reply:
By the way all charts which were published contains `values.yaml` inside:
`https://agones.dev/chart/stable`
For example this one: https://storage.googleapis.com/agones-chart/agones-1.4.0.tgz

Reply:
If there is a developer only terraform config - I think this should be put in put in ./build, not in ./examples.

I'm sure users would also find value in having local only versions as well, so it's likely they may use this as reference and reuse.

Reply:
Thanks for an update, that makes a lot of sense. I will move files according to their purpose.